src/test/java/io/jenkins/plugins/gitlabserverconfig/servers/GitLabServerTest.java

```diff
@@ -0,0 +1,65 @@
+package io.jenkins.plugins.gitlabserverconfig.servers;
```

Comment:
Please, specify the commit text according to its contents

Reply:
Also, this commit can be merged with [this one](https://github.com/jenkinsci/gitlab-branch-source-plugin/pull/131/commits/1b8cb9df8a74a2bc38b55642026882162aa7f050)